cmd/kuberhealthy/kuberhealthy.go

```diff
@@ -361,8 +367,14 @@ func (k *Kuberhealthy) monitorExternalChecks(ctx context.Context, notify chan st
 		// if a khcheck has been deleted, then we signal for change and purge it from the knownSettings map.
 		for mapName := range knownSettings {
 			var existsInItems bool // indicates the item exists in the item listing
-			for _, i := range l.Items {
-				itemMapName := i.Namespace + "/" + i.Name
+
+			for _, kc := range khChecks.Items {
+				khCheck, err := convertUnstructuredKhCheck(kc)
+				if err != nil {
+					continue
```

Comment:
add logging here because were deciding to do something based on an error that was returned